src/control/mod.rs

```diff
@@ -562,7 +562,10 @@ pub trait Device: super::Device {
 
         let prop_val_set = PropertyValueSet {
             prop_ids: unsafe { mem::transmute(prop_ids) },
-            prop_vals: unsafe { mem::transmute(prop_vals) },
+            prop_vals: unsafe {
+                #[allow(clippy::useless_transmute)] // TODO: Resolve clippy warning
+                mem::transmute(prop_vals)
```

Comment:
Should be fixed in `main` now: https://github.com/Smithay/drm-rs/pull/118/files#diff-cbe991c0ba9a31e65dfbb22e683b3a981f61154335714b511f54c8cd8d95ff12L565-R571